library/Notifications/Widget/Timeline/Entry.php

```diff
@@ -9,12 +9,14 @@
 use Icinga\Module\Notifications\Widget\TimeGrid;
 use ipl\Html\HtmlElement;
 use ipl\Html\Text;
+use ipl\Html\ValidHtml;
 use ipl\Web\Widget\Icon;
 
 class Entry extends TimeGrid\Entry
 {
     /** @var Member */
     protected $member;
+    protected $flyoutContent;
```

Comment:
Please add:

* a type hint (`?ValidHtml`)
* a docstring
* an empty line before it